streams-designer/README.md

```diff
@@ -45,15 +45,31 @@ These are the IBM Bluemix services required to run this tutorial:
 Cloud Object Storage instance was not created beforehand, you will have the option of adding one.
 ![Create New Project in Watson Studio](create-new-project-watson-studio.png) 
 
-## Create a Streams flow
+## Option 1: Import the Streams flow file
+1. Create a new Streams flow under the `Add to project` drop-down list.
+![Select Streams flow](streams-flow-watson-studio.png)
+1. On the **New Streams Flow** screen, Select **From file**.  Drag and drop [Cloudant_to_Db2_streams_flow.stp](Cloudant_to_Db2_streams_flow.stp)
+under the **File** section.
+The Streaming Analytics service drop-down box should already be filled in.  Press **Create**.
+![Import Streams flow](import-streams-flow-watson-studio.png)
+1. Edit the streams flow by clicking the pencil icon.
+![Open Streams Designer Editor](open-streams-designer-editor.png)
+1. Select the Code operator to open the code editor in the right pane. Replace the lines with `username`, `password`, and
+`account` with your Cloudant credentials.
+![](code-operator-streams-designer.png)
+1. [Setup the Db2 operator.](#setup-the-db2-operator)
+
+## Option 2: Create a new Streams flow
 1. Create a new Streams flow under the `Add to project` drop-down list.
 ![Select Streams flow](streams-flow-watson-studio.png)
 1. On the **New Streams Flow** screen, add a name and optional description for the project. 
 The Streaming Analytics service drop-down box should already be filled in.  
 Select the **Manually** box, then press **Create**.
 ![Create Streams flow](create-streams-flow-watson-studio.png)
 
-## Create operators in the Streams Designer editor
+## Create the Code operator
+**Note:** Skip this section if you imported the streams file.
```

Comment:
Instead of the note to skip this, I think it would be better to make this more steps under option 2.

Reply:
Moved steps in 28577cb.